tutor/resources/images/icons/exercise-exclude-card-view-action-transparent.svg

```diff
@@ -0,0 +1,9 @@
+</defs>
+</svg>
```

Comment:
where'd this SVG come from?  It looks like it's a SVG that continues a rasterized image.  It'd be better to use one that has paths.

Reply:
I see the image embedded here. I got it from figma, let me take a look.